diff --git a/lldb/cmake/modules/LLDBStandalone.cmake b/lldb/cmake/modules/LLDBStandalone.cmake
index d3e43cfac9f2..801be7dc9b0f 100644
--- a/lldb/cmake/modules/LLDBStandalone.cmake
+++ b/lldb/cmake/modules/LLDBStandalone.cmake
@@ -139,7 +139,7 @@ endif()
 if (NOT TARGET clang-resource-headers)
   include(GetClangResourceDir)
   get_clang_resource_dir(LLDB_EXTERNAL_CLANG_RESOURCE_DIR
-    PREFIX "${Clang_DIR}/../../../")
+    PREFIX "${LLVM_INSTALL_PREFIX}")
 
   if (NOT EXISTS ${LLDB_EXTERNAL_CLANG_RESOURCE_DIR})
     message(FATAL_ERROR "Expected directory for clang-resource-headers not found: ${LLDB_EXTERNAL_CLANG_RESOURCE_DIR}")
